contrib/test/run_test_vectors.sh

```diff
@@ -2,7 +2,7 @@
 
 # this assumes fd_ledger has already been built
 
-set -x
+set -xe
```

Comment:
-e will fail if any of the scripts fail, definitely useful to avoid regressions

Reply:
This will probably cause the test on the icelake machine to fail since it runs with asan and ubsan enabled